test-apps/ui-test-app/config/rush-project.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush-project.schema.json",
+  "operationSettings": [
+    { "operationName": "build", "outputFolderNames": ["lib"] },
+    { "operationName": "build:ci", "outputFolderNames": ["lib"] }
+  ]
+}
```

Comment:
I guess it doesnt really matter, but ui-test-app do not have `build` and `build:ci` scripts, but rather `buildapp` and `build:ci:optional` (So we dont build those when we dont need to), should we change the operation name, or for test apps we would like to "disableBuildCacheForOperation", like for the presentation-test-app ?

Same situation/question for appui-test-app.

Reply:
Yep will give that a look, thanks. The only reason I disabled the cache for the presentation-test-app was that it uses symlinks which Rush does not support.

Reply:
It seems ui-test-app isn't built by rush directly? If so, then the cache wouldn't even be applicable.

Reply:
Indeed, ui-test-app (and appui-test-app) were removed from the rush build to reduce the build time as not everyone needed those and the build took a significant time (on top of that, I'm not sure the built version were never really used as we "start" them with webdev server...)